packages/backend/src/modules/user/user.controller.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable class-methods-use-this */
+import express from 'express'
+import userService from './user.service'
+import userMapper from './user.mapper'
+
+class UserController {
+  async create(req: express.Request, res: express.Response): Promise<void> {
+    const createdUser = await userService.create(req.body)
```

Comment:
To keep it consistant we should specify an interface for a command and dto. Take a look at [this PR](https://github.com/kacper-cyra/camp-trello-clone/pull/113/files/77b2c384973bb2ecfa47f9cdf3baa067f7e440ee#diff-da68fbcd7f0eea91befb6535128e0c5f43dea6ef744e288a870abc9095fb5e54).

Reply:
agree add DTO/Command for request

Reply:
The `UserService.create()` function already takes an argument of type `CreateUserDto`, I'm not sure what else is supposed to be added here?